backend/scripts/test_ai_model_selection.py

```
@@ -0,0 +1,172 @@
+验证不同AI任务是否使用了正确的模型
+"""
+
+import asyncio
+import logging
+import sys
+from pathlib import Path
+
+# 添加项目根目录到Python路径
+sys.path.insert(0, str(Path(__file__).parent.parent))
+
+from app.services.ai.chat_service import ChatService, TEMPLATE_MODEL_MAPPING
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)